jobs/release.Jenkinsfile

```diff
@@ -275,6 +282,39 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
             }
         }
 
+        if (brew_profile) {
+            stage('Brew Upload') {
+                def tag = pipecfg.streams[params.STREAM].brew_tag
+                for (arch in basearches) {
+                    def state = false
+                    def version = (params.VERSION).replaceAll("-", ".")
+                    def nvr = "rhcos-${arch}-${version}-0"
+                    state = shwrapCapture("""
+                    coreos-assembler koji-upload search \
+                        --nvr ${nvr} \
+                        --keytab "/run/kubernetes/secrets/brew-keytab/brew.keytab" \
+                        --owner ${brew_principal} \
+                        --profile ${brew_profile}
+                    """)
+                    // Check if no Brew upload was done yet
+                    // State 1 means brew build complete
```

Comment:
can we link here to documentation that describes what the different states could be?